tests/utils/test_dependency_specification.py

```diff
@@ -69,12 +71,38 @@
         ),
         (
             'requests [security,tests] >= 2.8.1, == 2.8.* ; python_version < "2.7"',
-            {
-                "name": "requests",
-                "markers": 'python_version < "2.7"',
-                "version": ">=2.8.1,<2.9.0",
-                "extras": ["security", "tests"],
-            },
+            (  # allow several equivalent versions to make test more robust
+                {
+                    "name": "requests",
+                    "markers": 'python_version < "2.7"',
+                    "version": ">=2.8.1,<2.9",
+                    "extras": ["security", "tests"],
+                },
+                {
+                    "name": "requests",
+                    "markers": 'python_version < "2.7"',
+                    "version": ">=2.8.1,<2.9.0",
+                    "extras": ["security", "tests"],
+                },
+                {
+                    "name": "requests",
+                    "markers": 'python_version < "2.7"',
+                    "version": ">=2.8.1,<2.9.dev0",
+                    "extras": ["security", "tests"],
+                },
+                {
+                    "name": "requests",
+                    "markers": 'python_version < "2.7"',
+                    "version": ">=2.8.1,<2.9.0.dev0",
+                    "extras": ["security", "tests"],
+                },
+                {
+                    "name": "requests",
+                    "markers": 'python_version < "2.7"',
+                    "version": ">=2.8.1,!=2.8.*",
```

Comment:
I don't understand how this works. "Greater or equal 2.8.1 and not matching 2.8.*"?

Reply:
Good catch, it doesn't. 😉 It has to be `">=2.8.1,==2.8.*"`. (Right now it's just not relevant because neither version of poetry-core generates this string.) Fixed.